buildenv/jenkins/Jenkinsfile

```diff
@@ -73,19 +73,16 @@ pipeline {
         }
     }
     post {
-    	failure {
-    		slackSend channel: '#testing', color: '#FFFF00', message: "Jenkins test job failed: ${env.JOB_NAME} ${env.BUILD_NUMBER} (<${env.BUILD_URL}|Open>)"
-    	}
-    	unstable {
-    		slackSend channel: '#testing', color: '#FFFF00', message: "Test job unstable. Some tests failed: ${env.JOB_NAME} ${env.BUILD_NUMBER} (<${env.BUILD_URL}|Open>)"
-    	}
-    	
     	always {
  			step([$class: "TapPublisher", testResults: "**/*.tap"])
             archiveArtifacts artifacts: '**/*.tap', fingerprint: true
             junit allowEmptyResults: true, keepLongStdio: true, testResults: '**/work/**/*.xml, **/junitreports/**/*.xml'
             archiveArtifacts artifacts: '**/work/**/*.xml, **/junitreports/**/*.xml', fingerprint: true
+            script {
+            	if (params.TARGET == 'test_systemtest') {  
+	       	 		sh 'tar -zcvf openjdk-systemtest-results.tar.gz $WORKSPACE/openjdk-test/TestConfig/test_output_*'
+		    	}    
+	    	}  
```

Comment:
I think it is true that TARGET is only ever a single target, but please confirm, it is never a comma-separated list of targets passed in?

Reply:
I believe it will always be a single target, at least at the level of this Jenkins job. We may invoke this Jenkins script with various different targets, but it should not be a coma separated list passed into this. @sophia-guo - can you please confirm?